### PR TITLE
Add two packages for Debian

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds the `python3-distutils` and `python3-setuptools` packages for Debian-based OSs.  These packages no longer get installed automatically as dependencies for Debian Testing (currently Debian Trixie) and therefore not for Kali Linux either.

## 💭 Motivation and context ##

This was noticed because of the GH Actions failures that suddenly appeared in cisagov/ansible-role-assessment-tool#55.  See #60 for more details.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: molecule/default/tests/test_default.py [hunk 1]
```diff
@@ -13,7 +13,12 @@
 
 def test_pip3(host):
     """Test that the appropriate pip3 packages were installed."""
-    debian_pkgs = ["python3-pip", "python3-dev"]
+    debian_pkgs = [
+        "python3-pip",
+        "python3-dev",
+        "python3-distutils",
+        "python3-setuptools",
```
Comment (dav3r) [inline molecule/default/tests/test_default.py]: Alphabetize:
```suggestion
        "python3-dev",
        "python3-distutils",
        "python3-pip",
        "python3-setuptools",
```

**Review by dav3r — APPROVED** [review]
LGTM with one minor note. 👍

**Review by dv4harr10 — APPROVED** [review]

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Thanks for the fix 👍 However, please update the README to mirror other changes in this PR around platforms no longer supported:
https://github.com/cisagov/ansible-role-pip/blob/7155d3c43bfcd2c808253036628bc860b5c0f382/README.md?plain=1#L18

**Comment by jsf9k** [comment]
Please see commit 1fca4a0.

**Comment by jsf9k** [comment]
> Thanks for the fix 👍However, please update the README to mirror other changes in this PR around platforms no longer supported:
> 
> https://github.com/cisagov/ansible-role-pip/blob/7155d3c43bfcd2c808253036628bc860b5c0f382/README.md?plain=1#L18

Please see commit 1fca4a0.  I will amend [cisagov/ansible-role-python](https://github.com/cisagov/ansible-role-python) similarly tomorrow.

**Review by mcdonnnj — APPROVED** [review]

**Pull request merged**